src/ResourceManager/Sql/Commands.Sql/Database/Cmdlet/StopAzureSqlDatabaseActivity.cs

```diff
@@ -0,0 +1,70 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.Database.Cmdlet
+{
+    [Cmdlet(VerbsLifecycle.Stop, "AzureRmSqlDatabaseActivity", SupportsShouldProcess = true)]
+    public class StopAzureSqlDatabaseActivity : AzureSqlDatabaseActivityCmdletBase
+    {
+        /// <summary>
+        /// Gets database activity in an Elastic Pool
+        /// </summary>
+        /// <returns>List of Database activity</returns>
+        protected override IEnumerable<AzureSqlDatabaseActivityModel> GetEntity()
+        {
+            return ModelAdapter.ListDatabaseActivity(this.ResourceGroupName, this.ServerName, this.ElasticPoolName, this.DatabaseName, this.OperationId);
+        }
+
+        /// <summary>
+        /// No user input to apply to model
+        /// </summary>
+        /// <param name="model">Model retrieved from service</param>
+        /// <returns>The model that was passed in</returns>
+        protected override IEnumerable<AzureSqlDatabaseActivityModel> ApplyUserInputToModel(IEnumerable<AzureSqlDatabaseActivityModel> model)
+        {
+            return model;
+        }
+
+        /// <summary>
+        /// No changes to persist to server
+        /// </summary>
+        /// <param name="entity">The output of apply user input to model</param>
+        /// <returns>The input entity</returns>
+        protected override IEnumerable<AzureSqlDatabaseActivityModel> PersistChanges(IEnumerable<AzureSqlDatabaseActivityModel> entity)
+        {
+            return ModelAdapter.CancelDatabaseActivity(this.ResourceGroupName, this.ServerName, this.ElasticPoolName, this.DatabaseName, this.OperationId);
+        }
+
+        /// <summary>
+        /// Entry point for the cmdlet
+        /// </summary>
+        public override void ExecuteCmdlet()
+        {
+            if (!ShouldProcess(
```

Comment:
This shouldn't be necessary. It is done in [AzureSqlCmdletBase](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Sql/Commands.Sql/Common/AzureSqlCmdletBase.cs#L124)

Reply:
removed